jwql/instrument_monitors/common_monitors/cosmic_ray_monitor.py

```diff
@@ -192,16 +195,72 @@ def get_cr_mags(self, jump_locs, jump_locs_pre, rateints, jump_data, jump_head):
         Returns:
         -------
 
-        mags: list
-            A list of cosmic ray magnitudes corresponding to each jump.
+        mags: numpy.array
+            A histogram of cosmic ray magnitudes, from -65536 to 65536, with the number of
+            cosmic rays of each magnitude.
 
         """
+        mag_bins = np.arange(65536*2+1, dtype=np.int32) - 65536
+        mags = np.zeros_like(mag_bins, dtype=np.int32)
```

Comment:
I did that because I didn't think it was possible to create a numpy array that didn't use a numpy data type (but `sqlalchemy` apparently doesn't have a conversion function to turn numpy datatypes into sql datatypes, hence the later casting). It's entirely possible that I'm just wrong about that.

Reply:
From what I've read, np.int is identical to python's int these days. So maybe you can just do np.zeros_like(mag_bins, dtype=int)

Reply:
Cool, looks like it was handled in [7c4bdc7](https://github.com/spacetelescope/jwql/pull/921/commits/7c4bdc782ba31c4eff3cfef1ae53b88b3543b06e) @bhilbert4. Thanks @york-stsci